### DISCUSSION TITLE
Laser_scan_matcher_node shows warning message when I run gazebo simulation

### DISCUSSION THREAD

**Discussion by ak-ty** [HumanSupportRobot/QandA#50] [Q&A] [answered]
When I run gazebo simulation for hsrb (e.g `roslaunch hsrb_gazebo_launch hsrb_apartment_world.launch`), 
the following warning messages are continuously displayed on the console

> [ WARN] [1684307967.856404575, 173.100000000]: Could not get base to fixed frame transform, Lookup would require extrapolation 0.100000000s into the future.  Requested time 172.975000000 but the latest data is at time 172.875000000, when looking up transform from frame [base_footprint] to frame [odom]
> [ WARN] [1684307967.993263905, 173.200000000]: Could not get base to fixed frame transform, Lookup would require extrapolation 0.100000000s into the future.  Requested time 173.075000000 but the latest data is at time 172.975000000, when looking up transform from frame [base_footprint] to frame [odom]
> [ WARN] [1684307968.117681671, 173.300000000]: Could not get base to fixed frame transform, Lookup would require extrapolation 0.100000000s into the future.  Requested time 173.175000000 but the latest data is at time 173.075000000, when looking up transform from frame [base_footprint] to frame [odom]

This message is output by "/hsrb/laser_scan_matcher_node".

Does anyone know how to avoid this warning message ?

Thank you.

**Comment by hsr-admin** [comment, marked as answer]
The same problem occurred with the latest scan_tools.

Please clone the previous version of scan_tools ([0.3.3](https://github.com/CCNYRoboticsLab/scan_tools/tree/0.3.3)) and build it to try.

**Comment by ak-ty** [comment]
Thank you for sharing your solution.

By using scan_tool(0.3.3), the warning message does not appears in my environment.
